fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/PaloAuth.java

```diff
@@ -1076,20 +1096,25 @@ private void createRoleInternal(String role, boolean isReplay) throws DdlExcepti
 
     // drop role
     public void dropRole(DropRoleStmt stmt) throws DdlException {
-        dropRoleInternal(stmt.getQualifiedRole(), false);
+        dropRoleInternal(stmt.getQualifiedRole(), stmt.isSetIfExists(), false);
     }
 
     public void replayDropRole(PrivInfo info) {
         try {
-            dropRoleInternal(info.getRole(), true);
+            dropRoleInternal(info.getRole(), false, true);
         } catch (DdlException e) {
             LOG.error("should not happened", e);
         }
     }
 
-    private void dropRoleInternal(String role, boolean isReplay) throws DdlException {
+    private void dropRoleInternal(String role, boolean ignoreIfNonExists, boolean isReplay) throws DdlException {
         writeLock();
         try {
+            if (ignoreIfNonExists && roleManager.getRole(role) == null) {
+                LOG.info("role non exists, ignored to drop role: {}, is replay: {}", role, isReplay);
```

Comment:
```suggestion
                LOG.debug("role non exists, ignored to drop role: {}, is replay: {}", role, isReplay);
```

Reply:
I notice that, the following code also include LOG.info, maybe we also correct it ?

Reply:
yes, I copy and paste these log statements from nearby. Noticed that `LOG.debug` and `LOG.info` are mixed.